frontend/src/components/Challenges/ChallengeItemCard.tsx

```diff
@@ -25,12 +25,11 @@ const ChallengeItemCard: React.FC<ChallengeItemCardProps> = ({
   const ipfsImageUrl = convertIpfsImageUrl(imageUrl);
   const [loading, setLoading] = useState(true);
   const router = useIonRouter();
-
   const handleClick = () => {
     console.log(tokenId, mintingContractAddress, ipfsImageUrl, "all of these?");
     if (!loading) {
       router.push(
-        `/challenge/${tokenId}?&contractAddress=${mintingContractAddress}&imageUrl=${ipfsImageUrl}`
+        `/mint/1111/challenge/${tokenId}?&contractAddress=${mintingContractAddress}&imageUrl=${ipfsImageUrl}`
```

Comment:
What is the 1111 used for here?

Reply:
Seems like this route does not work, just checked locally and I can not navigate to a challenge when I click on one from the discover page. Not sure if you had something else in mind but I added back the Challenge routes in App.tsx like this:

`<Route path="/challenges" render={() => <Challenges />} exact />
      <Route path="/challenge/:id" render={() => <Challenge />} exact />`

And changed this line to `router.push(
        `/challenge/${tokenId}?&contractAddress=${mintingContractAddress}&imageUrl=${ipfsImageUrl}`
      );`

Reply:
fixed now